src/io/jesi/backpack/fn.cljc

```diff
@@ -102,3 +103,15 @@
        and returns true if ANY individual predicates return true, else return false."}
   or-fn
   (partial apply-predicates (comp boolean some)))
+
+(defn ->comparator
+  "Returns a comparator where values returning from a value function are compared against"
+  [val-fn]
+  (fn [x y]
+    (compare (val-fn x) (val-fn y))))
+
+(def ^{:arglists '([pred coll])
+       :doc      "Returns true if any item in coll returns true for pred, otherwise false"}
+  any?
+  (comp boolean some))
```

Comment:
What if we want to do `(any? false? [false])`? This will currently return false, it should be true.
I think we want to do a nil check.

Reply:
NVM, I was confused by `some`

Reply:
`(any? false? [false])` will be true.

```
user=> ((comp boolean some) false? [false])
true
user=>